### DISCUSSION TITLE
Phoenix 1.6 removes the need for node, do I update the elixir+phoenix devcontainer?

### DISCUSSION THREAD

**Discussion by krainboltgreene** [community/community#6166] [Codespaces]
Hi, so there's a devcontainer called "elixir, phoenix, postgres" that installs nodejs. However Phoenix 1.6, which just released, removes the need for nodesjs.

Should I submit a new devcontainer without nodejs, remove nodejs from the old one, or write variation logic?

**Comment by matthewisabel** [comment]
Hey, thanks for checking in on what the best path forward is. I created an issue in that repo that has the sample devcontainers: https://github.com/microsoft/vscode-dev-containers/issues/1105.

@2percentsilk and @Chuxel can suggest what we should do here.

**Reply by Chuxel** [reply]
I @'d in the community owners who made a suggestion. Feel free to discuss with them further on this or other adjustments you think would be good to do.  We'll @ them in to whatever PR you raise to get a thumbs up and then merge.

**Comment by NALLELUNA** [comment]
> Hi, so there's a devcontainer called "elixir, phoenix, postgres" that installs nodejs. However Phoenix 1.6, which just released, removes the need for nodesjs.
> 
> Should I submit a new devcontainer without nodejs, remove nodejs from the old one, or write variation logic?
